great_expectations_cloud/agent/agent.py

```diff
@@ -361,8 +364,8 @@ def _handle_event_as_thread_exit(
                 )
         else:
             status = build_failed_job_completed_status(error)
-            LOGGER.info(traceback.format_exc())
-            LOGGER.info(
+            LOGGER.error(traceback.format_exc())
+            LOGGER.error(
```

Comment:
I think this should stay as `info`. While the job errored, that's not necessarily (and not likely) an error in the agent. More likely it's something like incorrect credentials, attempting to test a non-existent table or column, etc.